src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -1293,6 +1294,7 @@ class SelectClauseElementSegment(BaseSegment):
     """An element in the targets of a select statement."""
 
     type = "select_clause_element"
+    reflow_pre_space = "true"
```

Comment:
Did you mean to make this a boolean value, as elsewhere in the file?
```suggestion
    reflow_pre_space = True
```

Reply:
Certainly did!